### ISSUE TITLE
Websocket error 400

### ISSUE THREAD

**Issue by VivianoRiccardo** [localtunnel/localtunnel#369]
So, i tried to proxy the request on my local websocket server and tried to connect to localtunnel with a websocket connection and this error occurred: "failed: Error during WebSocket handshake: Unexpected response code: 400". I have read the other issues regarding websockets but still i could not understand if localtunnel supports websocket or not.

**Comment by ECorreia45** [comment]
I am running into same issue right now. Anyone on this?

**Comment by VivianoRiccardo** [comment]
for what i saw there is no websocket implementation

**Comment by ECorreia45** [comment]
I ended up using ngrok...seems to load faster as well.

**Comment by VivianoRiccardo** [comment]
yes but ngrok doesn't work with websockets, so is useless for this

**Comment by ECorreia45** [comment]
My website has WebSocket and it is working fine

**Comment by tomleejumah** [comment]
anyone here who has tried Serveo?
